google_signin/lib/main.dart

```diff
@@ -41,13 +41,15 @@ class AppState extends State<App> {
     FirebaseAuth _auth = FirebaseAuth.instance;
     try {
       googleSignIn = GoogleSignIn();
-      GoogleSignInAccount googleSignInAccount = await googleSignIn.signIn();
-      GoogleSignInAuthentication gauth =
-          await googleSignInAccount.authentication;
-      FirebaseUser user = await _auth.signInWithGoogle(
-        accessToken: gauth.accessToken,
-        idToken: gauth.idToken,
+      GoogleSignInAccount googleUser = await googleSignIn.signIn();
+      final GoogleSignInAuthentication googleAuth =
+      await googleUser.authentication;
+      final AuthCredential credential = GoogleAuthProvider.getCredential(
+        accessToken: googleAuth.accessToken,
+        idToken: googleAuth.idToken,
```

Comment:
I haven't run this app yet, but can you make sure the variables are not renamed.

`GoogleSignInAccount googleSignInAccount`

`final GoogleSignInAuthentication gauth`

Reply:
Apologies, I was in hurry last time, just copy pasted the code from stack overflow. Fixed it now!